Library/Homebrew/os/mac/xcode.rb

```diff
@@ -143,7 +144,7 @@ def default_prefix?
         if version < "4.3"
           prefix.to_s.start_with? "/Developer"
         else
-          prefix.to_s.start_with? "/Applications/Xcode.app"
+          prefix.to_s == "/Applications/Xcode.app/Contents/Developer"
```

Comment:
While I agree that the check probably should be more specific, this change feels really out-of-place in this PR. It feels doubly weird to change this line but to leave the check two lines above unchanged.